doc/tutorials/lris_howto.rst

```diff
@@ -0,0 +1,485 @@
+*not* included in the wavelength solution.  The top-right panel shows the fit (red) to the
+observed trend in wavelength as a function of spectral pixel (blue crosses); gray circles
+are features that were rejected by the wavelength solution.  The bottom-right panel shows the
+fit residuals (i.e., data - model).
+
+What you hope to see in this QA is:
+
+ - On the left, many of the arc lines marked with green IDs
+ - In the upper right, an RMS < 0.1 pixels
+ - In the lower right, a random scatter about 0 residuals
+
+In addition, the script :ref:`pypeit-chk-wavecalib` provides a summary of the wavelength calibration
+for all the spectra. We can run it with this simple call:
+
+.. code-block:: bash
+
+    pypeit_chk_wavecalib Calibrations/WaveCalib_A_0_DET01.fits
+
+and it prints on screen the following (here truncated) table:
+
+.. code-block:: bash
+
+     N. SpatID minWave Wave_cen maxWave dWave Nlin     IDs_Wave_range    IDs_Wave_cov(%) mesured_fwhm  RMS
+    --- ------ ------- -------- ------- ----- ---- --------------------- --------------- ------------ -----
+      0    212  6835.4   8469.9 10121.3 1.610   43  6931.379 -  9925.919            91.1          2.7 0.027
+      1    265  6038.3   7674.1  9327.6 1.612   45  6336.179 -  9227.030            87.9          2.7 0.025
+      2    335  7169.5   8831.4 10485.5 1.611   39  7427.339 - 10472.923            91.8          2.7 0.348
+      3    384     0.0      0.0     0.0 0.000    0     0.000 -     0.000             0.0          0.0 0.000
+      4    474     0.0      0.0     0.0 0.000    0     0.000 -     0.000             0.0          0.0 0.000
+
+See :ref:`pypeit-chk-wavecalib` for a detailed description of all the columns.
+Note that the slits with ``SpatID`` 384 and 474 have all the values set to 0.0. This is because
+these slits are actually alignment boxes and they are not reduced by the pipeline.
+
+Tilts
+:::::
+
+Wavelength tilts are measured performing a 2D fit to the traced arc lines.
+There are several QA files written for the 2D fits. One example is
+``QA/PNGs/Arc_tilts_2d_A_0_DET01_S0212.png``:
+
+.. image:: ../figures/lris_arc2d.png
+  :scale: 20%
+
+Each horizontal line of black open circles identifies a traced arc line.
+Red circles shows the points that were rejected in the 2D fitting, and the
+black points show the model predicted position.  The text across the top of
+the figure gives the RMS of the 2D wavelength solution, which should be less than 0.1 pixels.
+
+The 2D fit for the wavelength tilts can also be inspected using the script :ref:`pypeit_chk_tilts`,
+which shows a :ref:`tiltimg` image in a `ginga`_ or `matplotlib`_ window with the
+traced and 2D fitted tilts over-plotted. Here is an example:
+
+.. code-block:: bash
+
+    pypeit_chk_tilts Calibrations/Tilts_A_0_DET01.fits
+
+.. image:: ../figures/lris_tilts.png
+   :scale: 40%
+
+This shows a zoom-in of a :ref:`tiltimg` image in a `ginga`_ window with overlaid
```

Comment:
We should probably comment why some lines aren't colored anything

Reply:
done